.github/workflows/merge-check.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build:
     name: Build project
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
As a follow-up, could we create a linux CI job?
- So that people on linux can still build this project
- We also observed flakiness in macos-latest in the main detekt repo

Reply:
created: https://github.com/detekt/sarif4k/pull/8